frontend/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@material-tailwind/react": "^2.1.9",
     "@monaco-editor/react": "^4.6.0",
+    "@next/font": "^14.2.7",
```

Comment:
I think `next/font` is already imported from `next`, do we need to include this import? If not we can remove it and delete changes in `frontend/package-lock.json`